config/samples/gitops/fluxcd-application-kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: gitops.kubesphere.io/v1alpha1
+kind: Application
+metadata:
+  name: chengleqi-test-kus
+  namespace: my-devops-projecthmhx2
+spec:
+  kind: fluxcd
+  fluxApp:
+    spec:
+      source:
+        sourceRef:
+          kind: GitRepository
+          name: fluxcd-gitee-repo
+          namespace: my-devops-projecthmhx2
+      config:
+        kustomization:
+          - destination:
+              kubeConfig:
```

Comment:
same above

Reply:
It can only reconcile local cluster flux app for now if we don't do some thing like below manually.
`kubectl create secret generic prod-kubeconfig --from-file=value.yaml=./kubeconfig`

And I'll write `multi-cluster-controller` in the next few days to reconcile the kubeconfig. Then I'll add the sample kubeconfig.